cli/command/image/build_test.go

```diff
@@ -24,6 +24,8 @@ import (
 
 func TestRunBuildResetsUidAndGidInContext(t *testing.T) {
 	skip.IfCondition(t, runtime.GOOS == "windows", "uid and gid not relevant on windows")
+	skip.IfCondition(t, runtime.GOOS == "openbsd", "gotestyourself/fs not working right on openbsd")
```

Comment:
Do you have a stack trace for the error you hit here?

Reply:
Good question. This looks to be related to needing elevated permissions. If run as root, the tests pass.

As an unprivileged user:
```
$ go test -tags daemon -v github.com/docker/cli/cli/command/image
=== RUN   TestRunBuildResetsUidAndGidInContext
--- FAIL: TestRunBuildResetsUidAndGidInContext (0.01s)
        Error Trace:    file.go:62
                        build_test.go:38
        Error:          Received unexpected error:
                        chown /tmp/test-build-context-503524866/foo: operation not permitted
=== RUN   TestRunBuildDockerfileFromStdinWithCompress
--- FAIL: TestRunBuildDockerfileFromStdinWithCompress (0.03s)
        Error Trace:    build_test.go:70
                        client_test.go:121
                        build.go:390
                        build_test.go:98
        Error:          Received unexpected error:
                        lchown /tmp/test-build-compress-dest607440761/foo: operation not permitted
        Error Trace:    build_test.go:108
        Error:          Not equal: 
                        expected: []string{".dockerfile.69701b88c18326ae964e", ".dockerignore", "foo"}
                        received: []string{"foo"}
                    
                        Diff:
                        --- Expected
                        +++ Actual
                        @@ -1,4 +1,2 @@
                        -([]string) (len=3) {
                        - (string) (len=32) ".dockerfile.69701b88c18326ae964e",
                        - (string) (len=13) ".dockerignore",
                        +([]string) (len=1) {
                          (string) (len=3) "foo"
...
```